master/buildbot/manhole.py

```diff
@@ -13,6 +13,7 @@
 #
 # Copyright Buildbot Team Members
 from __future__ import print_function
+from builtins import str
```

Comment:
```str``` in python3 is the same as ```unicode``` in python2. ```str``` in python2 has no equivalent in python3.```from builtins import str``` backports the py3 version of ```str``` (unicode) to py2